module/support/logging/DataLogging/src/DataLogging.hpp

```diff
@@ -41,8 +41,13 @@ namespace module::support::logging {
                 std::string binary;
                 /// The threshold of bytes where after this we split the file
                 uint64_t split_size{};
+
+                uint64_t max_size{};
```

Comment:
"What even is this?"  -  Kip

Reply:
smells funky lol. Just zero initialise them.